build-support/gen_notice.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/python3
+# This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+from collections import defaultdict
+import glob
+import re
+import sys
+
+def gen_combined_notice(licenses_paths, outpath, gen_all):
+    all = defaultdict(lambda: [None, None])
+    for licenses_path in licenses_paths.split(','):
+        notices = glob.glob(licenses_path + '/NOTICE-*.txt')
+        for n in notices:
+            name = re.search(r'NOTICE-(.+)\.txt', n).group(1)
```

Comment:
should check file name error

Reply:
Why will there be a name error? If so, this script will fail as expected.